collector/crio/metrics.csv

```diff
@@ -0,0 +1,64 @@
+Name,Description,Unit,DataType,Attributes
```

Comment:
Do we have other metrics out of crio that aren't the default from the Go runtime (https://github.com/cri-o/cri-o/blob/main/tutorials/metrics.md)?